### PR TITLE
Add data-qa attributes to help city-specific e2e testing

### PR DESCRIPTION
**Pull request by apeltola**

### CODE REVIEW

File: frontend/src/citizen-frontend/applications/editor/AdditionalDetailsSection.tsx [hunk 1]
```diff
@@ -61,6 +61,7 @@ export default React.memo(function AdditionalDetailsSection({
 
           <FixedSpaceColumn spacing="xs">
             <ExpandingInfo
+              data-qa="diet-info"
```
Comment (mikkopiu) [inline frontend/src/citizen-frontend/applications/editor/AdditionalDetailsSection.tsx:64]: Nit: `diet-expanding-info` would be more in line with the other new attributes.

File: frontend/src/lib-components/molecules/ExpandingInfo.tsx [hunk 2]
```diff
@@ -95,7 +98,9 @@ export default function ExpandingInfo({
               size="s"
             />
 
-            <InfoContainer>{info}</InfoContainer>
+            <InfoContainer data-qa={`${dataQa ?? ''}-text`}>
```
Comment (mikkopiu) [inline frontend/src/lib-components/molecules/ExpandingInfo.tsx:101]: Wouldn't it be better to leave the whole attribute out if `dataQa` is not set?

**Review by mikkopiu — CHANGES_REQUESTED** [review]

**Review by reynders — APPROVED** [review]

**Review by mikkopiu — APPROVED** [review]
LGTM, pushing to CI

**Comment by mikkopiu** [comment]
@apeltola some lint errors in CI, see status links

**Pull request merged**